doc/news/astropy.rst

```diff
@@ -0,0 +1,4 @@
+**Changed:**
+
+* Changed maximum astropy version to 7.x.
```

Comment:
```suggestion
* Changed upper version bound for astropy from <=7 to <=8.
```

Reply:
Maybe we should change it to `<9` to be consistent with how we indicate all other packages.